react/utils/resolveSlidesNumber.js

```diff
@@ -0,0 +1,18 @@
+function resolveSlidesNumber(perPage) {
+  let result
+  if (typeof perPage === 'number') {
+    result = perPage
+  } else if (typeof perPage === 'object') {
+    result = 1
+    if (window) {
```

Comment:
perhaps add a comment on what this function is trying to achieve (i.e. what cases it covers, maybe with an example)